components/table/table-col-sort-button.js

```diff
@@ -67,13 +67,24 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 			button::-moz-focus-inner {
 				border: 0;
 			}
+			button:hover {
+				background-color: var(--d2l-color-gypsum);
+			}
+			:host([has-sibling]) button {
+				--d2l-table-cell-padding: 0;
+
+				margin-block: -0.3rem -0.3rem;
+				margin-inline: -0.3rem 0.3rem;
+				padding: 0.3rem;
+			}
 			:host([has-sibling]) button:hover {
-				text-decoration: underline;
+				border-radius: 0.2rem;
 			}
 			:host(:not([has-sibling])) button {
+				border-radius: var(--d2l-sortable-button-border-radius);
```

Comment:
Was there a case that caused this to need to move here?

Reply:
The spec for the interim styles have a border radius of 0.2rem but in default table the border radius should be 0